targets/esp8266/source/jerry_extapi.c

```diff
@@ -31,12 +31,11 @@
 #define __UNSED__ __attribute__((unused))
 
 #define DELCARE_HANDLER(NAME) \
-static bool \
-NAME ## _handler (const jerry_api_object_t * function_obj_p __UNSED__, \
-                  const jerry_api_value_t *  this_p __UNSED__, \
-                  jerry_api_value_t *        ret_val_p __UNSED__, \
-                  const jerry_api_value_t    args_p[], \
-                  const jerry_api_length_t   args_cnt)
+static jerry_value_t \
+NAME ## _handler (const jerry_value_t  function_obj_p __UNSED__, \
+                  const jerry_value_t  this_p __UNSED__, \
```

Comment:
Please remove the `_p` suffix. They are not pointers anymore. ex.: `func_obj_val` and `this_val`